vividus-plugin-web-app/src/test/java/org/vividus/selenium/TimeoutConfigurerTests.java

```diff
@@ -35,6 +36,8 @@
     private static final int ASYNC_SCRIPT_TIMEOUT = 30;
     private static final TimeUnit ASYNC_SCRIPT_TIMEOUT_TIMEUNIT = TimeUnit.SECONDS;
 
+    private static final Duration DURATION_PAGE_LOAD_TIMEOUT = Duration.ofSeconds(30L);
```

Comment:
not critical, so mostly FYI: my personal recommendation is to do not move values to constants which are used in 1 test only, I recommend to keep local variable in a test method, the reason is simple: when number of tests is growing, the number of constants is growing and it becomes harder and harder to read the test code and to understand its logic